blazor/heatmap-chart/working-with-data.md

```diff
@@ -9,6 +9,11 @@ documentation: ug
 
 # Working with data in Blazor HeatMap Chart Component
 
+This section shows how to bind data to the HeatMap component using the video below.
```

Comment:
This section shows how to bind data to the HeatMap component. The below video demonstrates the same.